app/lib/katello/logging.rb

```diff
@@ -9,6 +9,40 @@ def self.time(message, data: {}, logger: Rails.logger, level: :info)
       data_string = data.map { |k, v| "#{k}=#{v}" }.join(' ')
 
       logger.send(level, "#{message} #{data_string}")
+      "#{message} #{data_string}"
```

Comment:
Why return this string?

Reply:
@jturel It's a feature! :smile: 

When I was using it in the console, I would have to go searching in the log file to find the message.  Returning it guarantees that it's displayed on the screen.  (I now see that the log is also displayed.. not sure if I just missed that before)

Reply:
Since the log message is displayed in the console if you're using this method there, I'd prefer to not return this string as that's not the intent of the method.

Reply:
@jturel Ok, removed